### DISCUSSION TITLE
Website Taxonomy

### DISCUSSION THREAD

**Discussion by troyhunt** [HaveIBeenPwned/ux-rebuild#5] [Ideas]
The layout and navigation of the site is largely fine as it is, with one notable exception:

It's possible for people to purchase an API key and also to purchase access to search large domains. This makes 2 services, although they're tied into 1. Where we're finding confusion at present is someone signs up to the service, forgets all about it then gets an automated email from Stripe including the messaging "Go to the website to change or cancel your service". Uh, where, exactly?

I'd like to explore the idea of tying all the subscription info in together under a clear "subscription" menu that can be linked to directly from Stripe messages. It'd be cool to have one place where you see everything, but also to still keep "API" and "Domain Search" up there in the nav so they're not missed.

I'm sure there are many other ideas too, how else can we better improve the way information is laid out on the site?

**Comment by neilcaldwell** [comment]
I'd like to see more links at the footer of the page, there seems to be links added over time to About drop down, now at 9, it's a lot. I like the icons on the troyhunt.com website, much cleaner. 

Also, the "Notify Me" is a pop up on the home page, and a page in itself, on other pages, like from domain search. Perhaps it could be part of the main page?

**Comment by neilcaldwell** [comment]
The recapture boxes, should be updated to CloudFlare turnstile, so less obtrusive. https://www.cloudflare.com/products/turnstile/

**Comment by neilcaldwell** [comment]
PwnedWebsites could be sorted by date, with newest at top, with options to sort by size/alpha/date, perhaps a filter for type of data [email/IP/names/passwords/phone/etc]

**Reply by troyhunt** [reply]
Suggest you drop that one over here @neilcaldwell: https://github.com/HaveIBeenPwned/ux-rebuild/discussions/6

The list has also gotten way too long and is obviously only going to keep growing.

**Reply by neilcaldwell** [reply]
Moved @troyhunt [to #6](https://github.com/HaveIBeenPwned/ux-rebuild/discussions/6#discussioncomment-8639056)

**Comment by troyhunt** [comment]
> The recapture boxes, should be updated to CloudFlare turnstile, so less obtrusive. https://www.cloudflare.com/products/turnstile/

Totally agree, dedicated issue now created for it: https://github.com/HaveIBeenPwned/ux-rebuild/issues/7
